backend/dal/Helpers/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System.Text;
+
+namespace Pims.Dal.Helpers.Extensions
+{
+    /// <summary>
+    /// StringExtensions static class, provides extension methods for strings.
+    /// </summary>
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Extracts the first letter of each word from the text and returns the value.
+        /// </summary>
+        /// <param name="text"></param>
+        /// <param name="uppercase"></param>
+        /// <returns></returns>
+        public static string GetFirstLetterOfEachWord(this string text, bool uppercase = true)
+        {
+            var result = new StringBuilder();
+            var split = text.Split();
```

Comment:
You may need: StringSplitOptions.RemoveEmptyEntries here to handle double spaces or trailing spaces.